src/kOS/Communication/InterVesselManager.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using kOS.Safe.Encapsulation;
+using kOS.Safe.Serialization;
+using kOS.Serialization;
+using kOS.Safe;
+
+namespace kOS.Communication
+{
+    [KSPScenario(ScenarioCreationOptions.AddToAllGames | ScenarioCreationOptions.AddToExistingGames, GameScenes.FLIGHT, GameScenes.SPACECENTER, GameScenes.TRACKSTATION)]
```

Comment:
Was the goal here to make there always exist exactly one and only one instance of this class during the life of the KSP game?  Because I think there's a better simpler set of attributes for that, that we use for things like the telnet server, where you don't have to mention all the GameScenes one by one.  It looks like what you want for this class is for an instance of it to always exist and always respond to any load or save action anywhere.  I'm a little afraid that by mentioning explicit scenes it might prevent the saves from happening correctly if the save occurs on any other scene that wasn't mentioned in the list above.